test/functional/page_objects/visual_builder_page.ts

```diff
@@ -358,6 +358,12 @@ export class VisualBuilderPageObject extends FtrService {
     return await gaugeCount.getVisibleText();
   }
 
+  public async getGaugeColor(type: undefined | 'Inner' = undefined): Promise<string> {
```

Comment:
Am I right that we have only 2 cases 'Inner' and not-'Inner'. If it's true maybe we should use boolean instead?